api/protos/bootstrap/v1/bootstrap.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+
+package bootstrap;
+option go_package = "bootstrap/v1;bootstrapv1";
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/wrappers.proto";
+import "validate/validate.proto";
+
+
+// [#next-free-field: 4]
+message Bootstrap {
+    // xds-relay server settings.
+    Server server = 1;
+
+    // Configuration information about the control plane/upstream server.
+    Cluster control_plane_cluster = 2;
```

Comment:
Can we call this `Upstream origin_server` or `Upstream upstream_cluster`? Beyond the MVP, xds-relay could have the capabilities to be a standalone control plane solution and I'm worried this term might get conflated.

Reply:
renamed to origin_server